Makefile

```diff
@@ -57,6 +57,10 @@ gotidy:
 gofmt:
 	$(MAKE) for-all CMD="make fmt"
 
+.PHONY: golint
+golint:
+	$(MAKE) for-all CMD="make lint"
```

Comment:
Should this really be part of this PR?

Reply:
I used it to debug lint errors, I will split

Reply:
Done.